Mollie.Api/Client/Abstract/ITerminalClient.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.Threading.Tasks;
+using Mollie.Api.Models.List;
+using Mollie.Api.Models.Payment.Request;
+using Mollie.Api.Models.Payment.Response;
+using Mollie.Api.Models.Terminals;
+using Mollie.Api.Models.Url;
+
+namespace Mollie.Api.Client.Abstract
+{    /// <summary>
+     /// Calls in this class are documented in https://docs.mollie.com/reference/v2/terminals-api/overview
```

Comment:
This link doesn't work for me. Is there any documentation you can add to this PR?

Reply:
Hey Vincent, 
As soon as the documentation on Mollie's website is updated the link should work. The documentation I'm following is found at https://private.docs.molops.io/reference/v2/terminals-api/overview but this is a temporary link and a login is required. I just prepared the link for when the update is done at Mollie's side.